internal/contributebot/webhook/README.md

```diff
@@ -0,0 +1,27 @@
+# Contribute Bot Webhook Listener
+
+Contribute Bot Webhook Listener is a microservice that listens for
+[GitHub webhook][] events and publishes them to a [Cloud Pub/Sub][] topic.
+
```

Comment:
Sorry, I'm missing how this fits in with what Contribute Bot is intended to do.
Is there a separate component that will subscribe to the Pub/Sub topic and then react?
Why is the pub/sub indirection needed?

Reply:
There will be a separate worker that will run through the Pub/Sub topic. GitHub has a [10 second response time limit](https://developer.github.com/v3/guides/best-practices-for-integrators/#favor-asynchronous-work-over-synchronous) combined with a [5000 request/hour rate limit](https://developer.github.com/v3/#rate-limiting), so this adds buffering with the assumption that incoming events are bursty.

Reply:
Makes sense, can you add this as a comment somewhere, probably README.md?

Reply:
Done.